apps/studio/schemas/timeRange.tsx

```diff
@@ -0,0 +1,88 @@
+import type { DateTimeInputProps } from "sanity";
+import { defineField, defineType, set, unset } from "sanity";
+import ct from "countries-and-timezones";
+
+const timezones = ct.getAllTimezones();
+
+const CustomDateTimeInput = ({ onChange, value }: DateTimeInputProps) => {
+  return (
+    <input
+      type="datetime-local"
+      onChange={(event) => {
+        console.log(
+          event.currentTarget.value,
+          // This is probably pretty fragile, but seems to work well enough?
+          // Basically appending "+00:00" to instruct the Date contructor to interpret our string as UTC.
+          // The other option would be parsing the string and using Date.UTC().
+          // idk time zones are hard :/
+          new Date(event.currentTarget.value + "+00:00")
+        );
+        onChange(
+          event.currentTarget.value
+            ? set(new Date(event.currentTarget.value + "+00:00").toISOString())
+            : unset()
+        );
+      }}
```

Comment:
For readability purposes, it'd probably be better to parse the string and use `Date.UTC()`.

Reply:
I'm honestly not quite sure which one is better. Here's me parsing the date string:

```ts
const value = event.currentTarget.value;

const [year, month, day, hour, minute] = value
  .split(/[-T:]/)
  .map((string) => parseInt(string));

onChange(
  value
    ? set(
        new Date(Date.UTC(year, month, day, hour, minute)).toISOString()
      )
    : unset()
);
```

What do you think?

Reply:
I feel like this is better because you're explicitly stating that you're using UTC time.

Reply:
Sounds good!